samples/subsys/console/echo/README.rst

```diff
@@ -0,0 +1,43 @@
+.. zephyr:code-sample:: console_echo
+   :name: console echo
+   :relevant-api: console_api
+
+   Echo an input character back to the output using the Console API.
+
+Overview
+********
+
+This example shows how the :c:func:`console_getchar` function and
+:c:func:`console_putchar` function can be used to echo an input character
+back to the console.
+
+Requirements
+************
+
+UART console is required to run this sample.
```

Comment:
Similar to an earlier comment regarding being unnecessarily uart-specific, isn't the requirement rather: "A board that defines a `zephyr,console`."?